tableau_utilities/hyper/hyper.py

```diff
@@ -0,0 +1,129 @@
+import os
+import shutil
+from zipfile import ZipFile
+from tableauhyperapi import HyperProcess, Connection, Telemetry, CreateMode, TableDefinition, TableName, SqlType
+
+from tableau_utilities.tableau_file.tableau_file import TableauFileError
+from tableau_utilities.tableau_file.tableau_file import Datasource
+
+class HyperFile:
```

Comment:
This is a very rough cut and paste with a little duplicated code but it will give you the idea.

Reply:
Thanks Jay! Yeah thinking about the code change here, I imagine we may just want to make this a function rather than a class, and it would just take the `Datasource` class as an argument.
i.e. `add_empty_extract(Datasource)`